timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimestampBoundStore.java

```diff
@@ -207,15 +198,20 @@ SequenceAndBound forceAgreedState(long seq, @Nullable Long oldState) {
         if (seq <= PaxosAcceptor.NO_LOG_ENTRY) {
             return Optional.of(ImmutableSequenceAndBound.of(PaxosAcceptor.NO_LOG_ENTRY, 0L));
         }
-        PaxosResponses<PaxosLong> responses = PaxosQuorumChecker.collectQuorumResponses(
-                learners,
-                learner -> getLearnedValue(seq, learner),
-                QUORUM_OF_ONE,
-                executor,
-                PaxosQuorumChecker.DEFAULT_REMOTE_REQUESTS_TIMEOUT).withoutRemotes();
-        return Optional.ofNullable(Iterables.getFirst(responses.get(), null))
+
+        PaxosResponses<PaxosContainer<Optional<PaxosLong>>> responses =
+                learnerClient.getLearnedValue(seq, maybeValue -> PaxosContainer.of(maybeValue
+                        .map(PaxosValue::getData)
+                        .map(PtBytes::toLong)
+                        .map(ImmutablePaxosLong::of)));
```

Comment:
Note that this (I think) runs with the standard quorum, so we'd now wait for (generally) one more other node. In general looking back on it I have some concerns about the way we now use quorums in learners (right now we seek majority quorums, which seems unnecessary), though that's probably a separate discussion.

Reply:
Yeah, I went back and forth on this one, I didn't really want to include a quorum size parameter because that would make the batching logic really awkward and we'd be lying somewhat re passing in a quorum size.

I suppose we could special this method and say `getLearnedValueQuorumOfOne` or something. If anything will change in a later PR.

Reply:
Yeah. I think in general the Learner at least for `getLearnedValue` and probably in general should probably be created with a quorum of 1 for reads; quorum doesn't really give you any additional correctness guarantees. Not a super-high priority, in any case.